product_docs/docs/epas/15/reference/sql_reference/02_data_types/04_date_time_types.mdx

```diff
@@ -5,22 +5,27 @@ legacyRedirectsGenerated:
+  - ../../../../../epas_compat_reference/02_the_sql_language/02_data_types/04_date_time_types #generated for docs/epas/reorg-role-use-case-mode
+source:
+   url: https://www.postgresql.org/docs/current/datatype-datetime.html
+   timestamp: 2023-09-20
 ---
 
 <div id="date_time_types" class="registered_link"></div>
 
-The following discussion of the date/time types assumes that the configuration parameter `edb_redwood_date` is set to `TRUE` whenever a table is created or altered.
 
-EDB Postgres Advanced Server supports the date/time types shown in the table.
+| Name                             | Native | Alias | Description                                                                                   |
+| -------------------------------- | -------| ----- | ---------------------------------------------------------------------------------------------- |
+| `DATE`                           | ✅      |      | Date and time, 8 bytes storage, 4713 BC to 5874897 AD range, and resolution 1 second.              | 
+| `INTERVAL DAY TO SECOND [(p)]`   | ✅      |      | Period of time, 12 bytes storage,  -178000000 years to 178000000 years range, and resolution 1 microsecond / 14 digits |
+| `INTERVAL YEAR TO MONTH`         | ✅      |      | Period of time, 12 bytes storage, -178000000 years to 178000000 years range, and resolution 1 microsecond / 14 digits. |
+| `TIMESTAMP [(p)]`                | ✅      |      | Date and time, 8 bytes storage, 4713 BC to 5874897 AD range, and resolution 1 microsecond. |
+| `TIMESTAMP [(p)] WITH TIME ZONE` | ✅      |      | Date and time with time zone, 8 bytes storage, 4713 BC to 5874897 AD range, and resolution 1 microsecond.  |
+
 
-| Name                             | Storage size | Description                  | Low value        | High value      | Resolution                |
-| -------------------------------- | ------------ | ---------------------------- | ---------------- | --------------- | ------------------------- |
-| `DATE`                           | 8 bytes      | Date and time                | 4713 BC          | 5874897 AD      | 1 second                  |
-| `INTERVAL DAY TO SECOND [(p)]`   | 12 bytes     | Period of time               | -178000000 years | 178000000 years | 1 microsecond / 14 digits |
-| `INTERVAL YEAR TO MONTH`         | 12 bytes     | Period of time               | -178000000 years | 178000000 years | 1 microsecond / 14 digits |
-| `TIMESTAMP [(p)]`                | 8 bytes      | Date and time                | 4713 BC          | 5874897 AD      | 1 microsecond             |
-| `TIMESTAMP [(p)] WITH TIME ZONE` | 8 bytes      | Date and time with time zone | 4713 BC          | 5874897 AD      | 1 microsecond             |
+## Overview
+
+The following discussion of the date/time types assumes that the configuration parameter `edb_redwood_date` is set to `TRUE` whenever a table is created or altered.
```

Comment:
@nidhibhammar, @josh-heyer, @djw-m - These first four paragraphs have no corollary in the PG docs. Should they be left as is?

Reply:
Yes - these cover EPAS support for Oracle-syntax interval literals (as evidenced by the predicate on `edb_redwood_date`).